alertmanager_gchat_integration/app.py

```diff
@@ -41,31 +42,55 @@ def healthz():
     return ''
 
 
+def post_request(url: str, post_request_data):
+    post_request_details = requests.post(url, json=post_request_data)
+
+    if post_request_details.ok:
+        LOGGER.info('Alert message posted successfully.')
+    else:
+        LOGGER.error(f'Alert message failed to be posted: \n {post_request_data}')
+        raise Exception(f'Alert message failed to be posted. {post_request_details.status_code} - {post_request_details.text}')
+
 @app.route('/alerts', methods=['POST'])
 def post_alerts():
     """ Parses the given message and post it to the indicated channel. """
 
     # Get room name and validate.
     room_name = request.args.get('room', '')
     if room_name == '':
-        LOGGER.debug('Room <%s> not given as parameter.')
+        LOGGER.error('Room <%s> not given as parameter.', room_name)
         abort(400)
 
     # Check if room is defined in configuration.
     if room_name not in CONFIG['app']['room']:
-        LOGGER.debug('Room <%s> not present in configuration file.')
+        LOGGER.error('Room <%s> not present in configuration file.', room_name)
         abort(404)
 
-    # Notification URL.
     notification_url = CONFIG['app']['room'][room_name]['notification_url']
-    if 'origin' in CONFIG['app']['notification']:
-        origin = CONFIG['app']['notification']['origin']
-    else:
-        origin = os.environ.get('HOSTNAME', 'Unknown')
+    group_alerts = CONFIG['app']['room'][room_name].get('group_alerts', True)
+    group_alerts_truncate_after = CONFIG['app']['room'][room_name].get('truncate_after', 20)
+    origin = CONFIG['app']['notification'].get('origin', os.environ.get('HOSTNAME', 'Unknown'))
+
+    if USE_CARDS and not group_alerts:
```

Comment:
If I don't mistake myself, the if should be:

`if not USE_CARDS and group_alerts:`

But maybe I'm wrong?